test/functional/lua/vim_spec.lua

```diff
@@ -861,4 +861,23 @@ describe('lua stdlib', function()
     eq({}, exec_lua[[return {re1:match_line(0, 1, 1, 7)}]])
     eq({0,3}, exec_lua[[return {re1:match_line(0, 1, 0, 7)}]])
   end)
-end)
+
+  it('vim.defer_fn', function()
+    exec_lua [[
+    vim.g.test = 0
+    vim.defer_fn(function() vim.g.test = 1 end, 5)
+    vim.cmd("sleep 10m")
+    ]]
+    eq(1, exec_lua[[return vim.g.test]])
+  end)
+
+  it('vim.region', function()
+    helpers.insert(helpers.dedent( [[
+    text tααt tααt text
+    text tαxt txtα tex
+    text tαxt tαxt
+    ]]))
+    eq({5,15}, exec_lua[[ return vim.region(0,{1,5},{1,14},'v',true)[1] ]])
```

Comment:
Would you add another region(eg. 'V') test?

Reply:
I intentionally stuck with a simple test since it's not easy to compare nested tables in the form that `region` is supposed to return. To quote Bartleby: "I prefer not to." (And would leave that for subsequent PRs.)

Reply:
justin said he might end up changing a few bits around for some of this anyway, so it felt better to leave that to the subsequent PR.